src/binder/src/Shared/Trackers/BoundParentTracker.lua

```diff
@@ -29,6 +29,12 @@ function BoundParentTracker.new(binder, child)
 		end
 	end))
 
+	self._maid:GiveTask(self._binder:GetClassAddedSignal():Connect(function(_, instance)
+		if self._child.Parent == instance then
```

Comment:
We should probably use self._binder:Observe(instance) on the parent instead for performance. But this works, and the binder is already being listened to for the above area. Rx is definitely preferred here for pref long term.